features/networking/sdn.feature

```diff
@@ -299,4 +299,4 @@ Feature: SDN related networking scenarios
       | curl | <%= cb.pod1_ip%>:8080 |
     Then the step should succeed
     And the output should contain "Hello OpenShift"
-    """
+    """
```

Comment:
please check the end of this line. seems there is redundent character

Reply:
@zhaozhanqi Seems okay to me. See under "view file"